Add max_pixels option.

wenbinc-Bin opened the pull request:
Add max_pixels option, so that vllm resize large images.

czhu15 commented on scripts/start_gaudi_vllm_server.sh:
```diff
@@ -189,6 +189,7 @@ python3 -m vllm.entrypoints.openai.api_server \
     --trust-remote-code \
     --seed 2025 \
     --distributed_executor_backend "${dist_backend}" \
+    --mm_processor_kwargs "max_pixels=1003520" \
```
> If I change it to configurable, what option flag do you suggest I use?

how about just reuse -e 
cc @yangulei

yangulei replied:
If you want to pass `--mm_processor_kwargs "max_pixels=1003520"` to the vllm server, just pass `-e "--mm_processor_kwargs max_pixels=1003520"` to the `start_gaudi_vllm_server.sh`. We don't need to change the code.

wenbinc-Bin replied:
We should better set a default value in script because user may doesn't know this options is needed.

wenbinc-Bin replied:
I use **-e** opton, but with a default value for **mm-processor-kwargs**. How about this implementation?

wenbinc-Bin committed:
Fix large image bug on qwen3-vl

When input attn_mask and seq_len is large, FusedSDPA fail.

Because large input image doesn't use pad, attn_mask is set to
None for large input image to avoid this error.

Signed-off-by: Chen, Wenbin <wenbin.chen@intel.com>
